mobile/src/main/java/ch/epfl/sdp/mobile/application/chess/ChessFacade.kt

```diff
@@ -45,8 +45,8 @@ class ChessFacade(private val auth: Auth, private val store: Store) {
   }
 
   /**
-   * Fetches a [Flow] of [List] of [Match]s that a certain [Profile] has going on with any other
-   * player (or even himself)
+   * ======= >>>>>>> display-active-games Fetches a [Flow] of [List] of [Match]s that a certain
```

Comment:
Looks like a VCS issue.

```suggestion
   * Fetches a [Flow] of [List] of [Match]s that a certain
```